src/alert_rules/prometheus/zookeeper_metrics.rules

```diff
@@ -164,11 +173,11 @@ groups:
     labels:
       severity: warning
     annotations:
-      summary: "Zookeeper have too high ephemeral znodes count."
+      summary: "Zookeeper has too high ephemeral znodes count."
 
   - alert: Zookeeper Unrecoverable Errors
     expr: increase(unrecoverable_error_count{juju_charm!=".*"}[2m]) > 0
     labels:
       severity: high
     annotations:
-      summary: "Zookeeper have unhandled Exception"
\ No newline at end of file
+      summary: "Zookeeper has unhandled Exception"
```

Comment:
**typo, non-blocking** Do we need to be thorough with `ZooKeeper` everywhere (i.e grep-ing and stuff)? If we do not care about annotations, ignore that